lib/core/unit_conversion.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:math';
+
+const _kiloFactor = 1e3;
```

Comment:
use the packages
https://pub.dev/packages/units_converter
https://pub.dev/packages/fling_units

Reply:
this file is gone, units will be handled later on (another PR ?)